.prettierignore

```diff
@@ -8,6 +8,7 @@ github-actions/lock-closed/main.js
 github-actions/commit-message-based-labels/main.js
 github-actions/slash-commands/main.js
 github-actions/post-approval-changes/main.js
+github-actions/org-file-sync/main.js
```

Comment:
Can we put this as a `local-actions`, or are we expecting this to run in other repositories as well? I guess I don't know a concrete use-case yet, but seems useful 😄